source/tools/kubernetes/dask-operator.md

```diff
@@ -226,7 +226,7 @@ spec:
 ```
 
 For the scheduler pod we are also setting the `rapidsai/base` container image, mainly to ensure our Dask versions match between
-the scheduler and workers. We also disable Jupyter and ensure that the `dask-scheduler` command is configured.
+the scheduler and workers. We ensure that the `dask-scheduler` command is configured.
```

Comment:
I think there is something missing in the yaml file that actually disables Jupyter, but it's not clear to me. Please correct if I'm wrong.